addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2088,7 +2088,19 @@ msgctxt "#470"
 msgid "Credits"
 msgstr ""
 
-#empty strings from id 471 to 473
+#: xbmc/video/dialogs/GUIDialogVideoInfo.cpp
+
+msgctxt "#471"
+msgid "Banner"
+msgstr ""
+
+msgctxt "#472"
+msgid "Poster"
+msgstr ""
+
+msgctxt "#473"
```

Comment:
I think we could reuse label 21371 "Thumbnail" here. I also just noticed that we're quite inconsistent in label usage.While we use "thumbnail" most of the times, there are quite a lot usages of "thumb". I'll file a PR to get rid of "thumb", or are there any objections?

could you also please add a usage and contextual comment on each label? Contextual note could be something like: `#. Name of a artwork type`